packs/wechat/actions/lib/wechatutil/wechatclient.py

```diff
@@ -0,0 +1,24 @@
+from lib.wechatutil.sendmessages import send_msg
+from lib.wechatutil.wechatrequest import WechatRequest
+from lib.wechatutil.message import *
```

Comment:
The "import *" part seems to be causing some problems with tests. Can we make this more specific?